packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -233,6 +273,9 @@ export const makeVaultKit = (
 
   // call this whenever anything changes!
   const updateUiState = async () => {
+    if (!uiUpdater) {
+      return;
+    }
```

Comment:
No because notifications are always the latest state and so it can easily be sent later. the first thing we do after setting up a notifier is send it the current state.